draft-ietf-jsonpath-base.md

```diff
@@ -277,7 +273,7 @@ The JSONPath tool in question should:
 JSONPath expressions always apply to a Value in the same way
 as XPath expressions are used in combination with an XML document.
 Since a Value is anonymous, JSONPath uses the abstract name `$` to
-refer to the entire Value ("root node") of the argument.
+refer to the root node of the argument.
```

Comment:
The definition is not a replacement for the exposition in the text.

Reply:
Please explain why.  The root node is by definition the entire argument.

Reply:
In fact the root node is not a node initially, as it lacks a name by nature. Similar to your formulation, the root node formally becomes a node by always getting the name '$' assigned by convention. 

Your formulation

`... refer to the root node of the argument`

somehow contradicts your comment

`The root node is by definition the entire argument.` 

and also somehow confuses below  by ...

`... executed against a value, the *argument* ...`

BTW the definition above  is cristal clear, as the *argument* is the value of the *root node*.

Reply:
> In fact the root node is not a node initially - @goessner 

Not quite.  "Argument" is already defined as a value (not a node):

```
Short name for the value a JSONPath expression is applied to.
```

This means that "entire value of the argument" is redundant.  My change simply removes the redundancy.  "Root node of the argument" identifies the top-level location of the initial JSON value.